src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
This _should_ be ok. If anything, it'll be a tiny blip, but the rate of change on `sentry_project` should be low enough it should be fine.